src/components/tooltip/tooltip-pointer.style.js

```diff
@@ -18,7 +20,7 @@ const StyledTooltipPointer = styled.div`
       border-left: 8px solid transparent;
```

Comment:
Should all of these other hard coded px values also be updated?

Reply:
In the first step we replace the theme with tokens and don't create new CSS syntax. In the next step it will depend on how the component is described in DS.